### ISSUE TITLE
Changes to technology question and helper text changes

### ISSUE THREAD

**Issue by clairerye** [ebi-ait/hca-ebi-dev-team#327]
## Text changes:
1. Replace:"Technologies used to generate the data"

With: "What library preparation and/or imaging technique/s did you use to generate the data?"

2. replace: "Other Technologies"

With: "Other library preparation and/or imaging technique/s"

3. Replace: Please see <a href="https://ontology.dev.archive.data.humancellatlas.org" target="_blank">Ontology Lookup Service</a> for reference.

With correct formatting of the link. Also link should be:https://www.ebi.ac.uk/ols/ontologies/hcao

## Investigate:
1. Adding additional terms in the dropdown from http://purl.obolibrary.org/obo/OBI_0001686 as well
2. Can we remove the ontology curie and just have the label text
@mshadbolt I have put these two as investigate, did I forget anything else? @MightyAx if this relatively easy lets just do it. If its going to be complex, lets have a chat first.

**Comment by mshadbolt** [comment]
I actually think that providing the link to the ols isn't necessary. I think we should just be using the labels as text but not really make any explicit link to ontologies that the user can see

**Comment by clairerye** [comment]
Sounds good, so @mshadbolt the helper text should read 
"Start typing to select the technology. You can select multiple entries. Please note that if the search result is too large, not all options may be displayed." ?

**Comment by mshadbolt** [comment]
Is there control over how many options are displayed? I think having only 5 is a bit limited

I find that text kind of clumsy, I would probably write something like

> Search for each technique by searching and selecting as many as are applicable. If you cannot find it, please enter the name of the technology in the 'Other technologies' box below.

**Comment by clairerye** [comment]
updated the ticket to reflect the above

**Comment by MightyAx** [comment]
Please confirm: @clairerye / @mshadbolt
Replace:

> Start typing to select the technology. You can select multiple entries. Please note that if the search result is too large, not all options may be displayed. Please see Ontology Lookup Service for reference.


With:

> Search for each technique by searching and selecting as many as are applicable. If you cannot find it, please enter the name of the technology in the 'Other library preparation and/or imaging technique/s' box below.

**Comment by mshadbolt** [comment]
@gabsie do you have any input, I still don't actually like my suggested text, every time I read it it seems more confusing

**Comment by MightyAx** [comment]
**Select** each technique by searching and selecting as many as are applicable. If you cannot find it, please enter the name of the technique in the 'Other library preparation and/or imaging technique/s' box below.

**Comment by mshadbolt** [comment]
yeah I quite like enter! I think having search twice quite close was throwing the sentence off

**Comment by clairerye** [comment]
Reads well to me

**Comment by MightyAx** [comment]
What do we think of the helper text for Other technologies now, still okay?
![Screenshot 2020-09-22 at 10 21 55](https://user-images.githubusercontent.com/13031259/93865144-d1607700-fcbd-11ea-883d-81d7fa28df02.png)

**Comment by clairerye** [comment]
should we change 'technologies' to 'techniques'? @mshadbolt

**Comment by MightyAx** [comment]
> If you cannot find any of the techniques used to generate your data, please enter their names here.
?

**Comment by mshadbolt** [comment]
should we add something like 'in the search box above'

> If you cannot find any of the techniques used to generate your data _in the search box above_, please enter their names here.

**Comment by MightyAx** [comment]
Great, this is where we are then for now. @gabsie these are easy enough to change if you have any feedback
![Screenshot 2020-09-22 at 10 33 31](https://user-images.githubusercontent.com/13031259/93866121-2c469e00-fcbf-11ea-8585-f95c2b42c333.png)

**Comment by MightyAx** [comment]
Will explicitly callout that the additional terms in the dropdown( from http://purl.obolibrary.org/obo/OBI_0001686) are not yet deployed to dev but there is work being done on it.

**Comment by MightyAx** [comment]
Can we close this issue?
